### PR TITLE
TeamJ_contrib

### PR DESCRIPTION
**Pull request by UmaArunachalam8**

### CODE REVIEW

**Comment by smbryan** [comment]
Looks good. Thank you

File: wiki/simulation/NDT-Matching-with-Autoware.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+## Normal Distribution Transform (NDT)
```
Comment (smbryan) [inline wiki/simulation/NDT-Matching-with-Autoware.md:1]: Missing the header information.

**Comment by UmaArunachalam8** [comment]
Sorry about that! Added the header

**Comment by smbryan** [comment]
Merged! Thanks again

**Pull request merged**